prov/gni/src/gnix_av.c

```diff
@@ -526,8 +527,23 @@ static int gnix_av_insert(struct fid_av *av, const void *addr, size_t count,
 	return ret;
 }
 
-static int gnix_av_remove(struct fid_av *av, fi_addr_t *fi_addr, size_t count,
-			  uint64_t flags)
+DIRECT_FN int gnix_av_insertsvc(struct fid_av *av, const char *node,
+				const char *service, fi_addr_t *fi_addr,
```

Comment:
These might as well have STATIC also, since many will be implemented eventually.